test/hummingbot/client/command/test_balance_command.py

```diff
@@ -0,0 +1,62 @@
+import asyncio
+import unittest
+from typing import Awaitable
+from unittest.mock import patch, MagicMock
+
+from hummingbot.client.config.global_config_map import global_config_map
+from hummingbot.client.hummingbot_application import HummingbotApplication
+from test.mock.mock_cli import CLIMockingAssistant
+
+
+class BalanceCommandTest(unittest.TestCase):
+    @patch("hummingbot.core.utils.trading_pair_fetcher.TradingPairFetcher")
+    def setUp(self, _: MagicMock) -> None:
+        super().setUp()
+        self.ev_loop = asyncio.get_event_loop()
+        self.app = HummingbotApplication()
+        self.cli_mock_assistant = CLIMockingAssistant()
+        self.cli_mock_assistant.start()
+
+    def tearDown(self) -> None:
+        super().tearDown()
+        self.cli_mock_assistant.stop()
```

Comment:
Correct me if i'm wrong but I believe `super.tearDown()` should be called at the end instead.

Reply:
True. Fixed